src/common/reconciler.go

```diff
@@ -133,6 +138,11 @@ func (r *ServiceReconciler) handleService(registration opslevel_jq_parser.Servic
 	service, status := r.lookupService(registration)
 	switch status {
 	case serviceAliasesResult_NoAliasesMatched:
+		if viper.GetBool("disable-service-create") {
```

Comment:
Another place to put this check could be in `OpslevelClient.CreateServiceHandler` which is what I did originally, but it requires more changes.